src/main/java/edu/ucsb/cs156/rec/entities/RecommendationRequest.java

```diff
@@ -0,0 +1,37 @@
+package edu.ucsb.cs156.rec.entities;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+
+/**
+ * This is a JPA entity that represents a Recommendation Request
+ */
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Entity(name = "recommendationrequest")
+public class RecommendationRequest {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private long id;
+
+  private String professorName;
+  private String professorEmail;
+  private String requesterName;
```

Comment:
requesterName is insufficient, and I'd argue, unnecessary.

Instead, I suggest using a ManyToOne mapping to the user's table, like this one from proj-courses:

https://github.com/ucsb-cs156/proj-courses/blob/4bdc2674508428d03cb2dcd2e9077d75cdaf5a3f/src/main/java/edu/ucsb/cs156/courses/entities/PersonalSchedule.java#L24

To be honest, I'd do the same for professor.

Then you can access all fields in the user table (name, email, etc.) directly from the entity for a RecommendationRequest, for both the professor and the requester.